.lintstagedrc.json

```diff
@@ -1,3 +1,3 @@
 {
-    "**/*.{js,ts,md,json,sol,yml,yaml}": "prettier --write"
+    "**/*.{js,ts,md,json,sol,yml,yaml}": "yarn prettier --write"
```

Comment:
am I the only one having an issue?
Locally lintstaged can't find the prettier bin otherwise

Reply:
Ah, I may have prettier installed as a global package.. yarn prettier is better tho.